pwa/src/templates/templateParts/dependenciesTemplates/networkDependenciesTemplate/filters/NetworkDependenciesFiltersTemplate.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
```

Comment:
I'm wondering, we're building a complete new filters component here. However, we already have a filter component, the `LayerAccordionFiltersTemplate`.

Visually, they are an exact match (I think)?

The only difference would be their `handleClick` per item. 

Could this be as simple as to just rename the `LayerAccordionFiltersTemplate` to `LayerFiltersTemplate`, and implementing that one in both places?

Reply:
I see that in the current implementation (I think we've spoken about this, might be in an upcomming pull request) that we're using `badgeNumber` within the `LayerAccordionFiltersTemplate`, which would be the difference between these two.

I would suggest removing that `badgeNumber` altogether, and adding an optional `disabled` prop, which would make `LayerAccordionFiltersTemplate` less context-dependent _and_ make it implementable in the new context.